motioneye/utils.py

```diff
@@ -275,15 +275,22 @@ def make_str(s):
     if isinstance(s, str):
         return s
 
+    elif isinstance(s, bytes):
```

Comment:
Check out six.ensure_str() and six.ensure_text() https://github.com/benjaminp/six/blob/master/six.py#L901